Makefile

```diff
@@ -1,6 +1,6 @@
-# make cloud_edge_components
-.PHONY: cloud_edge_components
-cloud_edge_components:
+# make cloud-edge-components
+.PHONY: cloud-edge-components
+cloud-edge-components:
```

Comment:
what about `make all` so to `all` here?

Reply:
And then run `make all WHAT={path to component}` to build individual components.

Reply:
@m1093782566 No it will be very complicated to implement .. If we had many components as in kubernetes cmd folder(apiserver,scheduler,kubelet, etc.....)then there was a need to implement **make all WHAT={path to component}** ex: make all WHAT=cmd/kubelet GOFLAGS=-v to compile binaries and to run test cases for individual package....In our case we have only 2 components to build, And also **edge** and **cloud** are in different  folder location ...I think the approach we are following is fine...   cc @rohitsardesai83 @sids-b

Reply:
> In our case we have only 2 components to build

I am sure we will add more components in the cloud.

Reply:
> I am sure we will add more components in the cloud.
> ok then it makes sense

Reply:
![Screenshot from 2019-04-11 20-47-24](https://user-images.githubusercontent.com/39593583/55969627-c1a4cc80-5c9b-11e9-8ac4-cbb8a4da1c37.png)

Reply:
@m1093782566 -Please review